app/src/main/java/com/testtask/apptesttask/model/interactor/characters/CharactersInteractor.kt

```diff
@@ -1,15 +1,12 @@
 package com.testtask.apptesttask.model.interactor.characters
 
-import com.testtask.apptesttask.model.repository.MarvelRepository
-import javax.inject.Inject
+class CharactersInteractor {
 
-class CharactersInteractor @Inject constructor(var marvelRepository: MarvelRepository) :
-    Interactor {
-
-    override fun getCharacters(ts: String, apikey: String, hash: String) =
-            marvelRepository.getHeroesList(ts, apikey, hash)
+    fun getCharacters(ts: String, apikey: String, hash: String) {
```

Comment:
You shouldn't pass all these values through the chain call. This information will be included by `Repository`.